README.md

```diff
@@ -671,7 +693,41 @@ If Ddosify can't receive the response for a request, that step is marked as Fail
 | `(status_code == 200) \|\| (status_code == 201)`   | same as preceding one |
 | `regexp(body,\"[a-z]+_[0-9]+\",0) == \"messi_10\"`   | checks if matched result from regex is equal to "messi_10" |
 
+## Test-Wide Assertion
+
+Ddosify now supports test-wide assertions, allowing users to verify the success of their load tests based on response times and failure counts of iterations. With this feature, users can assert the percentile of response times and the failure counts of all the iterations in a test.
+
+Users can specify the required percentile of response times and failure counts in the configuration file, and the engine will compare the actual response times and failure counts to these values throughout the test continuously. According to users configuration test can be aborted or continue running until end. Check the [example config](https://github.com/ddosify/ddosify/blob/master/config_examples/config.json) to see how it looks.
+
+Note that functions and operators mentioned in Assertion section can be used test-wide assertions also.
```

Comment:
Note that functions and operators mentioned in Assertion section can be used test-wide assertions also.

⬇️ 

Note that the functions and operators mentioned in the [Step Assertion](https://github.com/ddosify/ddosify#assertion) section can also be utilized for the Test-Wide Assertion keywords listed below.